pkg/querier/stats/stats.proto

```diff
@@ -28,4 +28,6 @@ message Stats {
   uint32 sharded_queries = 5;
   // The number of split partial queries executed. 0 if splitting is disabled or the query can't be split.
   uint32 split_queries = 6;
+  // The number of index bytes fetched for the query
```

Comment:
I would clarify it's just from the store-gateway, because we query the index in ingesters too.